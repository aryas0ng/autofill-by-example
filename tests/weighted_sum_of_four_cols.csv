1,9,0,2,27
2,8,5,1,37
2,7,4,0,28
3,6,3,2,32
10,5,2,1,30
5,4,1,0,
4,3,0,9,
19,2,2,8,
2,1,1,7,
1,0,0,6,
9,4,6,5,
7,3,5,4,
4,2,4,3,
3,1,3,2,
12,0,2,1,
4,1,1,0,
1,0,0,2,
26,5,3,1,
8,4,2,0,
3,3,1,3,
3,2,0,2,
8,1,4,1,
4,0,3,0,
6,9,2,5,
5,8,1,4,
4,7,0,2,
3,6,6,1,
2,5,5,0,
1,4,4,2,
0,3,3,1,
3,2,2,3,
2,1,1,3,
1,0,0,4,
0,4,3,46,
6,3,2,5,
5,2,1,6,