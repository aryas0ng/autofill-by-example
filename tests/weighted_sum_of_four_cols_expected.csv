1,9,0,2,27
2,8,5,1,37
2,7,4,0,28
3,6,3,2,32
10,5,2,1,30
5,4,1,0,16
4,3,0,9,46
19,2,2,8,61
2,1,1,7,35
1,0,0,6,25
9,4,6,5,55
7,3,5,4,44
4,2,4,3,32
3,1,3,2,22
12,0,2,1,22
4,1,1,0,9
1,0,0,2,9
26,5,3,1,49
8,4,2,0,22
3,3,1,3,24
3,2,0,2,15
8,1,4,1,26
4,0,3,0,13
6,9,2,5,50
5,8,1,4,40
4,7,0,2,26
3,6,6,1,37
2,5,5,0,27
1,4,4,2,29
0,3,3,1,19
3,2,2,3,25
2,1,1,3,19
1,0,0,4,17
0,4,3,46,201
6,3,2,5,38
5,2,1,6,36